### PR TITLE
Add inlets-operator dev instructions

### PR DESCRIPTION
**Pull request by dirien**
Signed-off-by: Engin Diri engin.diri@mail.schwarz

- [ ] I have raised an issue to propose this change.

## Description
Add inlets-operator dev instructions for docker-desktop and rancher-desktop for linode and azure

## How Has This Been Tested?

no tests needed

## How are existing users impacted? What migration steps/scripts do we need?
no

## Checklist:

I have:

- [ ] updated the documentation and/or roadmap (if required)
- [x] read the [CONTRIBUTION](https://github.com/inlets/inlets/blob/master/CONTRIBUTING.md) guide
- [x] signed-off my commits with `git commit -s`
- [ ] added unit tests

### CODE REVIEW

File: docs/DEV.md [hunk 1]
```diff
@@ -1,5 +1,89 @@
 # Local dev with Go binary
 
+## Run the go binary with Azure and Rancher Desktop
```
Comment (alexellis) [inline docs/DEV.md:3]: I would put this at the end of the file as it's not the main usecase

**Comment by alexellis** [comment]
I'm not sure that we need all these examples such as "run as a go binary on X cloud"

It seems very repetitive, can we just tell them to run --help and leave one of the current examples like DigitalOcean?

Happy to keep your rancher/docker desktop note at the end, if you feel it's non-obvious.

File: docs/DEV.md (hunk 1 above)
Comment (dirien) [reply docs/DEV.md]: done :)

**Comment by dirien** [comment]
@alexellis: could be handy one time, for others. But feel free to add or not, i just wanted to retain some of the challanges i went through

File: docs/DEV.md [hunk 2]
```diff
@@ -79,3 +81,86 @@ go build && ./inlets-operator \
...
+
+Go to [Rancher Desktop](https://rancherdesktop.io/) and download and install Rancher Desktop.
+
+Select your kubernetes version and take care to choose the `rancher-desktop` as kubernetes context
+
+![img.png](images/rancher-desktop.png)
+
+Sign Up at [Azure Portal](https://azure.microsoft.com/en-us/free/search/).
```
Comment (alexellis) [inline docs/DEV.md:92]: We already have this at: https://docs.inlets.dev/#/tools/inlets-operator?id=inlets-operator-reference-documentation

File: docs/DEV.md [hunk 3]
```diff
@@ -79,3 +81,86 @@ go build && ./inlets-operator \
...
+ 
+export GOPATH=$HOME/go/
+go get -u github.com/inlets/inlets-operator
+cd $GOPATH/github.com/inlets/inlets-operator
+
+go get
+
+kubectl config use-context docker-desktop
+kubectl apply -f artifacts/crds/
+kubectl config view --flatten --minify > kubeconfig.yaml
```
Comment (alexellis) [inline docs/DEV.md:164]: I'm not sure this is necessary.

I run `--kubeconfig $HOME/.kube/config`, can you try it out?

**Comment by alexellis** [comment]
Hi @dirien,

I've left a few comments here and questions and tried to simplify everything in: fbe4503, could you take a look please?

**Comment by dirien** [comment]
drop it 😄

**Pull request closed without merging**